### PR TITLE
Added code to utility file to get data frame 

### PR DESCRIPTION
**Pull request by Brian-Hansen-Turton**
if file exists in resource folder use that, if not download from Kraggle and move to resource folder

### CODE REVIEW

File: src/imdb_utils.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import kagglehub
+import os
+import shutil
+
+resources_path = "../resources"
+file_name = "movie_metadata.csv"
+kaggle_data = "carolzhangdc/imdb-5000-movie-dataset"
+
+def aquireIMDbDataFrame():
```
Comment (36kp) [inline src/imdb_utils.py:10]: can we use more conventional names like `getIMDbDataFrame()`

File: src/imdb_utils.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    """
+    Aquires the IMDb data from a local file or from Kaggle if the file does not exist locally.
```
Comment (36kp) [inline src/imdb_utils.py:12]: Can we add  `Returns:` Section with `DataFrame: Pre-populated DataFrame object.`

File: src/imdb_utils.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    destination = os.path.join(resources_path, file_name)
+    copy_file(source, destination)
+
+
+def copy_file(source, destination):
+    """
+    Copies a file from source to destination.
+
+    :param source: The path of the source file to copy.
+    :param destination: The path where the file should be copied.
```
Comment (36kp) [inline src/imdb_utils.py:41]: Can we add `Raises:` section with `FileNotFoundError: if source file is not found on local destination path.` and `Exception: if an unknown error occurs.`

File: src/imdb_utils.py (hunk 1 above)
Comment (Brian-Hansen-Turton) [reply src/imdb_utils.py:10]: "get" would be used if the item is readily available without work, such as in memory store.
"acquire" would be used if the process of obtaining the item is involved, like a network request.
but if you want it to be "get" fine with me.   We can talk tonight

Comment (36kp) [reply src/imdb_utils.py:10]: I am ok with this rationale

**Review by 36kp — APPROVED** [review]
Looks good

**Pull request merged**